### ISSUE TITLE
UI Components - Popup broken

### ISSUE THREAD

**Issue by gwharton** [magento/magento2#33115]
### Preconditions (*)

1. Deploy 2.4.2-p1

### Steps to reproduce (*)

1. Go to home page from the front end
2. View Source in developer tool
3. Extract the static base url from the source code. In the below screenshot it is the boxed url
![image](https://user-images.githubusercontent.com/51679138/121185123-5d81f100-c883-11eb-8dde-2fee42fb4b13.png)
4. Append "/css/docs/popups.html" to the url (replacing /mage/calendar.css)
5. Navigate to the modified url and will see the page in the browser like-
![image](https://user-images.githubusercontent.com/51679138/121185394-a46fe680-c883-11eb-976b-e4b062fc3164.png)

### Expected result (*)

1. Try to launch any of the popup examples (Click on any of the Show popup buttons)

### Actual result (*)

1. Nothing happens when you click the button

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.4/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [x] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by ihor-sviziev** [comment]
@gwharton, could you provide some info on why you would have it working? AFAIK it designed to be shown locally only

**Comment by gwharton** [comment]
Hi @ihor-sviziev Am I misunderstanding the component. Is it not there for us to use? What do you mean "shown locally only". I'm trying to implement a popup, where the shopper clicks on a link and a popup is displayed with some additional info.

**Comment by ihor-sviziev** [comment]
@gwharton ah, sorry, you're right, seems like it's broken

**Comment by engcom-Alfa** [comment]
Hi @gwharton 
The link you have provided to reproduce steps .- https://testmachine/static/versionxxxxxxxxxxxx/frontend/Magento/luma/en_US/css/docs/popups.html is not working. It is a broken link.
Can you please update the steps by step procedure to reproduce exactly along with expected and actual. If you have any pre conditions like settings/ configurations to be done, kindly mention that as well.

**Comment by gwharton** [comment]
Hi there,

Sorry I wasn't clear what the url was. The example docs are installed in any Magento installation by default.

If you deploy Magento, then view source on the homepage.

Extract the static url base path, for example

https://www.example.com/static/version1622559438/frontend/Magento/luma/en_US

append /css/docs/popups.html  to this static path to get the url to the example pages/docs

for example https://www.example.com/static/version1622559438/frontend/Magento/luma/en_US/css/docs/popups.html 

www.example.com and version1622559438 will differ for your installation

**Comment by gwharton** [comment]
@magento give me 2.4-develop instance

**Comment by gwharton** [comment]
The link for the above instance being 

https://a97eb0fc29611655cf02f423c4052910-2-4-develop.instances.magento-community.engineering/static/version1622637021/frontend/Magento/luma/en_US/css/docs/popups.html

**Comment by engcom-Alfa** [comment]
Hi @gwharton 
The above link is forbidden to me again.

**Comment by gwharton** [comment]
Yeah, I think the deployed images from the bot only last a certain time. If you deploy your own image and use the instructions https://github.com/magento/magento2/issues/33115#issuecomment-852984284 to form your own URL.

**Comment by engcom-Alfa** [comment]
Hi @gwharton 
I am unable to view the popups.html in my view source code.
Can you please tell me from where to enable the popups or how you have done it.
You may please inform me if it is for any particular UI pages.
It will be very helpful if you are going to describe the step by step reproducible steps.

**Comment by gwharton** [comment]
- Deploy Magento 2.4-develop
- Go to home page
- View Source
- Extract the static base url from the source code. In the below screenshot it is the boxed url
![image](https://user-images.githubusercontent.com/30697781/120991082-2f17ef00-c779-11eb-8795-486d59332d03.png)
- Append "/css/docs/popups.html" to the url
- Browse to this url for the example documents for the UI Popup Component.

In my example above, the url would be. "https://reedling.lubefinder.com/dev1/static/version1622549209/frontend/Magento/luma/en_GB/css/docs/popups.html"
and it results in the following page
![image](https://user-images.githubusercontent.com/30697781/120990919-02fc6e00-c779-11eb-932b-779d78000e1d.png)

**Comment by engcom-Alfa** [comment]
As I re tested in the 2.4-develop using above commented reproducible steps.
Once after navigating to the _http://local.magento.com/static/version1623145471/frontend/Magento/luma/en_US/css/docs/popups.html_ (link as per mhy local configuration), Show popup link is not responsive.
We have many buttons that are being not responsive. Hence confirming the issue.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Alfa 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-42585` were created

**Issue Available**: @engcom-Alfa, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by engcom-Bravo** [comment]
Hi @gwharton,

Thanks for your reporting and collaboration.

We have verified the issue in Latest 2.4-develop instance and the issue is still reproducible.Kindly refer the screenshots.

<img width="1081" alt="Screenshot 2024-06-13 at 09 44 40" src="https://github.com/magento/magento2/assets/51680745/7efa09ba-dedf-427b-b400-98344a075629">


Nothing happens when you click the button


Hence Confirming the issue.

Thanks.

**Comment by Alexander-Khohklov** [comment]
@magento I am working on this
